Feature/extensions 2

mathewmeconry opened the pull request:
As requested in pull #15 I created a new pull

mathewmeconry committed:
adds serviceLevel parameter

mathewmeconry committed:
adds currency to configurable properties

mathewmeconry committed:
adds InstrId to transaction information

mathewmeconry committed:
adds xmlns and xmlschemalocation to configurable options

mathewmeconry commented:
@mtimofiiv can you check that one?

mtimofiiv commented:
This looks great, thanks @mathewmeconry! However there appears to be a merge conflict preventing me from merging, in `package-lock.json` of all places. The weird thing is that this repo does not have a lockfile at all.

Once it's resolved, I will merge this!

mathewmeconry committed:
removes package-lock.json and adds it to .gitignore

mathewmeconry committed:
Merge branch 'master' into feature/extensions_2

mathewmeconry commented:
Hey @mtimofiiv 
Thanks for the review.
I fixed the conflict and updated my branch with your master. 

You should consider adding the package-lock.json to the repo. Because then the package versions of your dependencies are fixed and don't have unwanted side effects when newer versions are downloaded.

mathewmeconry commented:
Hi @mtimofiiv 
Some time has passed and I wanted to check in if you can merge this now or if there are any other issues

mathewmeconry commented:
Hi @mtimofiiv
Some time has passed and I wanted to check in if you can merge this now or if there are any other issues
